Integrations/integration-Aella_StarLight.yml

```diff
@@ -0,0 +1,331 @@
+  name: severity
+  defaultvalue: "50"
+  type: 0
+  required: false
+- display: Trust any certificate (unsecure)
+  name: insecure
+  defaultvalue: "true"
+  type: 8
+  required: false
+script:
+  script: |-
+    import requests
+    import json
+    import time
+    import datetime
+
+    import urllib3
+    urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+    from requests.packages.urllib3.exceptions import InsecureRequestWarning
+    requests.packages.urllib3.disable_warnings(InsecureRequestWarning)
+
+    ''' GLOBAL VARS '''
+    URL = demisto.getParam('url') + '/aellaelastic'
+    USERNAME = demisto.getParam('credentials')['identifier']
+    PASSWORD = demisto.getParam('credentials')['password']
+    FETCH_INTERVAL = demisto.getParam('fetch_interval')
+    VALIDATE_CERT = not demisto.params().get('insecure', True)
+
+    ''' HELPER FUNCTIONS '''
+    def make_rest_call(end_point, username, password, action_result,
+        headers={}, params=None,
+        data=None, method='get'):
+        headers.update({'Accept': 'application/json'})
+        headers.update({'Content-Type': 'application/json'})
+
+        resp_json = None
+        request_func = getattr(requests, method)
+        if (not request_func):
+            action_result['status']='Unsupported method {}'.format(method)
+            return
+        try:
+            r = request_func(end_point, auth=(username, password),
+                data=json.dumps(data) if data else None,
+                headers=headers,
+                verify=VALIDATE_CERT,params=params)
+        except Exception as e:
+            action_result['status']='Server REST API exception {}'.format(e)
+            return
+
+        if r is not None:
+            action_result['r_text']=r.text
+            action_result['headers']=r.headers
+            action_result['r_status_code']=r.status_code
+
+        try:
+            resp_json = r.json()
+        except Exception as e:
+            action_result['status']='Json parse error {}'.format(
+                r.text.replace('{', ' ').replace('}', ' '))
+            return
+
+        if (200 <= r.status_code <= 399):
+            action_result['status'] = 'Success'
+        else:
+            action_result['status'] = 'Failed'
+
+        action_result['data'] = resp_json
+        return
+
+    ''' FUNCTIONS '''
+    def fetch_incidents_command():
+        fetch_interval = demisto.getParam('fetch_interval')
+        if fetch_interval is None:
+            fetch_interval = 15 * 60 # 15 minutes
+        else:
+            try:
+                fetch_interval = int(fetch_interval) * 60
+                if fetch_interval < 15 * 60:
+                    # Min is 15 minutes
+                    fetch_interval = 15 * 60
+            except:
+                fetch_interval = 15 * 60
+
+        cur_t = time.time()
+
+        lastRunTime = demisto.getLastRun()
+        #demisto.info('LastRun is {}'.format(lastRunTime))
```

Comment:
Do we need that line? If not, please remove.

Reply:
done